scss/application.scss

```diff
@@ -0,0 +1,142 @@
+
+p{
+
+  text-align: center;
+
+}
+
+.navbar-text {
+
+  font-size: 200%;
+  margin: 35px 20px 0 0;
+
+}
+
+.img-navbar{
+
+  margin: 0 0 0 20px;
+
+}
+
+.img_tech_logo{
+
+  height: 100px;
+  width: 100px;
+  margin: 0 0 0 60px;
+
+}
+
+
+.navbar{
+
+  height: 90px;
+  -webkit-box-shadow:0px 1px 1px;
+  -moz-box-shadow:0px 1px 1px;
+  box-shadow: 0px 15px 20px rgba(0,0,0,.1);
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.card-container{
+
+  margin: 30px 150px 150px 250px;
+  display: flex;
+
+}
+
+.wrapper{
+
+  display: flex;
+  width: 260px;
+  flex-flow: row;
+  margin: 0 0 0 20px;
+
+}
+
+.background-card{
+
+  border:1px solid #BED23C;
+  border-radius: 10px;
+  font-family:'Roboto';
```

Comment:
When you set a custom font family, always set a back up one: `font-family: 'Roboto', sans-serif;`

Reply:
By the way, I think you are not actually importing the font :thinking: